item-descriptions.lua

```diff
@@ -590,53 +596,54 @@ descriptions = {
     MILLSTONE = {
                 "A large grinding stone, used in a mill to produce flour, sugar, and",
                 "dyes much faster than a quern. It is too large to be operated by hand,",
-                "and must be powered for operation.  Millstones are made of stone."},
+                "and must be powered for operation."},
     ORTHOPEDIC_CAST = {
                 "Casts are made from plaster, and are used to keep broken bones in",
                 "place until they are healed. Applying a cast requires a bucket,",
                 "cloth and a water source."},
     PIPE_SECTION = {
                 "An enormous piece of pipe, it is a part of a screw pump."},
     QUERN = {   "A hand-operated mill for plants, grains, and seeds. It mills plants",
-                "much slower than a millstone. Must be built from stone."},
```

Comment:
Is this not true? http://dwarffortresswiki.org/index.php/DF2014:Quern says they must be built at a mason's workshop, and there's not a page for the quern item itself.

Of course, you could say that this isn't necessary in the description that's already built.

Reply:
I felt it was a bit superfluous because the player wouldn't be reading this description unless they'd already built a quern.